ts/state/selectors/conversations.ts

```diff
@@ -396,9 +407,75 @@ export const _getLeftPaneLists = (
   };
 };
 
+export const _getConversationRequests = (
+  lookup: ConversationLookupType,
+  comparator: (left: ReduxConversationType, right: ReduxConversationType) => number,
+  selectedConversation?: string
+): Array<ReduxConversationType> => {
+  const values = Object.values(lookup);
+  const sorted = values.sort(comparator);
+
+  const conversationRequests: Array<ReduxConversationType> = [];
+
+  for (let conversation of sorted) {
+    if (selectedConversation === conversation.id) {
+      conversation = {
+        ...conversation,
```

Comment:
instead of having two selectors both filtering all conversations, can you make one selector filtering them already, and use this `getSortedConversations` selector as args in `_getConversationRequests` and `_getLeftPaneLists` ?